fast_api_proj/app/services/user_services.py

```diff
@@ -0,0 +1,41 @@
+import redis
+import json
+from app.models.user import User, AdminUser, RegularUser
+from app.services.interfaces import IUserCreation, IUserProfileUpdate, IUserRetrieval
+
+
+class UserService(IUserCreation, IUserProfileUpdate, IUserRetrieval):
```

Comment:
This class and file name is very confusing. Should call it something with the word "redis" if it's coupled with that technology.